lib/src/main/kotlin/at/bitfire/ical4android/AndroidCalendar.kt

```diff
@@ -40,7 +41,9 @@ abstract class AndroidCalendar<out T: AndroidEvent>(
 ) {
 
     companion object {
```

Comment:
In AndroidEvent you moved the companion object to the end of the class. Should we not do that here and (other modified classes) as well?